telemetry/csharp/AwsToolkit.Telemetry.Events/AwsToolkit.Telemetry.Events.csproj

```diff
@@ -16,7 +20,11 @@
 	</PropertyGroup>
 	<ItemGroup>
 	  <PackageReference Include="log4net" Version="2.0.8" />
+	  <PackageReference Include="NuGet.Build.Tasks.Pack" Version="5.2.0"/>
```

Comment:
What does this do?

Reply:
When creating a package through msbuild, it was recommended to use this if we were to use it with a non SDK style project. I can remove this since our project is SDK styled.
https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package-msbuild#add-the-nugetbuildtaskspack-package